OpenRA.PathfinderTests/PathfinderTests.cs

```diff
@@ -0,0 +1,164 @@
+﻿using Moq;
+using NUnit.Framework;
+using OpenRA;
+using OpenRA.Mods.Common.Traits;
+using System;
+using System.Diagnostics;
+using System.Linq;
+
+namespace PathfinderTests
+{
+	[TestFixture]
+	public class PathfinderTests
+	{
+		private IMap BuildFakeMap(int mapWidth, int mapHeight)
+		{
+			var map = new Mock<IMap>();
+			map.SetupGet(m => m.TileShape).Returns(TileShape.Rectangle);
+			map.Setup(m => m.GetMapSize()).Returns(new int2(mapWidth, mapHeight));
+			map.Setup(m => m.Contains(It.Is<CPos>(pos => pos.X >= 0 && pos.X < mapWidth && pos.Y >= 0 && pos.Y < mapHeight))).Returns(true);
+
+			return map.Object;
+		}
+
+		private IWorld BuildFakeWorld(IMap map)
+		{
+			var world = new Mock<IWorld>();
+			world.SetupGet(m => m.IMap).Returns(map);
+			world.SetupGet(m => m.IWorldActor).Returns(new Mock<IActor>().Object);
+			return world.Object;
+		}
+
+		private static bool ValidPos(CPos pos, int mapWidth, int mapHeight)
+		{
+			return pos.X >= 0 && pos.X < mapWidth && pos.Y >= 0 && pos.Y < mapHeight;
+		}
+
+		[Test]
+		public void Find_path_on_flat_terrain_test()
```

Comment:
So it is in business world (I work full time as .net developer). Yet this convention changes when unit testing, as function names need to be language-fluent to fully understand what it does. I intended to follow that very same rule.

Not a big deal to switch it back to pascalcase if you wish so. Should I?